woodwork/column_schema.py

```diff
@@ -69,118 +69,100 @@ def __repr__(self):
         msg += ">"
         return msg
 
+    def _get_column_tags(self, semantic_tags, validate):
+        semantic_tags = _convert_input_to_set(semantic_tags, error_language='semantic_tags',
+                                              validate=validate)
 
-def _validate_logical_type(logical_type):
-    ltype_class = _get_ltype_class(logical_type)
-
-    if ltype_class not in ww.type_system.registered_types:
-        raise TypeError(f'logical_type {logical_type} is not a registered LogicalType.')
-    if ltype_class == Ordinal and not isinstance(logical_type, Ordinal):
-        raise TypeError("Must use an Ordinal instance with order values defined")
-
-
-def _validate_description(column_description):
-    if column_description is not None and not isinstance(column_description, str):
-        raise TypeError("Column description must be a string")
-
-
-def _validate_metadata(column_metadata):
-    if not isinstance(column_metadata, dict):
-        raise TypeError("Column metadata must be a dictionary")
-
-
-def _get_column_tags(semantic_tags, logical_type, use_standard_tags, validate):
-    semantic_tags = _convert_input_to_set(semantic_tags, error_language='semantic_tags',
-                                          validate=validate)
-
-    if use_standard_tags:
-        if logical_type is None:
-            raise ValueError("Cannot use standard tags when logical_type is None")
-        semantic_tags = semantic_tags.union(logical_type.standard_tags)
-
-    return semantic_tags
-
+        if self.use_standard_tags:
+            if self.logical_type is None:
+                raise ValueError("Cannot use standard tags when logical_type is None")
+            semantic_tags = semantic_tags.union(self.logical_type.standard_tags)
 
-def _is_col_numeric(col):
-    return 'numeric' in col.logical_type.standard_tags
+        return semantic_tags
 
+    def _is_numeric(self):
```

Comment:
Should these `_is_*` methods be properties instead since they never take arguments?

Also, maybe this is a separate issue/PR, but should these be public?

Reply:
I like the idea of them being properties! 

I've also been considering making them public. Should be pretty quick, so I'm fine adding that in here (I think EvalML requested it at some point?)

Reply:
@thehomebrewnerd made both those changes--can revert/change it if you're not a fan